terraform/README.md

```diff
@@ -0,0 +1,37 @@
+#Terraform Deployment
+
+We use terraform to create the spaces and services needed by the application and to deploy the application itself.
+
+We use terraform workspaces to keep the terraform state seperate for each environment.
+
+We store our state in an s3 bucket on the PaaS in the terraform space.
+
+Deploys will happen automatically from travis
+
+## Manual Deployment
+
+We should avoid manual deployments but the following instructions will allow you to do one.
+
+### Setup
+- install terraform (use `tfenv`)
+- install the latest [cloundfoundry provider](https://github.com/cloudfoundry-community/terraform-provider-cf/wiki#installations)
+
+### Deployment
+- use the correct branch
+  - develop for staging
+  - master for production
+- export environment variables for AWS credentials
```

Comment:
How would we find the AWS creds needed to access a bucket stored in the GPaaS AWS account?

Reply:
They are in 1password. I'll add that.